Makefile

```diff
@@ -0,0 +1,17 @@
+.PHONY: docs build
+
+setup-docs-ci:
+	pip install -e .[docs]
+	playwright install
+
+	cd docs && \
+	npm install --global yarn && \
+	npm install
+
+docs: 
+	cd docs && make generate_all
```

Comment:
one note: generate_all hits the live server AND the frontend app. This means every single commit will trigger a few pings. Probably fine, but a bit fragile

Reply:
What's the alternative?